app/views/users/profile.html.erb

```diff
@@ -127,7 +127,7 @@
           <% if current_user.following?(@profile_user) %>
             <a id="profile-unfollow-button" class="btn btn-primary" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
           <% else %>
-            <a id="profile-follow-button" class="btn btn-primary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%= @profile_user.id %>"><%= translation('users.profile.follow') %></a>
+            <a id="profile-follow-button" class="btn btn-secondary" data-method="post" data-remote="true" href="/relationships" data-params="followed_id=<%= @profile_user.id %>"><%= translation('users.profile.follow') %></a>
```

Comment:
@garimavs It's the unfollow button that should have a different color. You can use the class `btn-light`.

Reply:
Thanks for highlighting. I used the class "btn-secondary" to have a grey shade even when not hovering, but will update that too!